packages/yoga/src/Dropdown/web/Dropdown.jsx

```diff
@@ -31,92 +33,95 @@ const Wrapper = styled.div`
+    transform: translateY(-${input.height / 2 - 2}px);
+    transition-property: transform, font-size, color;
+    transition-duration: ${transition.duration[1]}ms;
+    transition-timing-function: cubic-bezier(${transition.timing[0].join()});
 
-    background-color: ${dropdown.selector.background};
-    border-radius: ${dropdown.selector.border.radius}px;
-    border-width: ${dropdown.selector.border.width}px;
-    border-style: solid;
-    border-color: ${
-      error ? colors.feedback.attention[1] : dropdown.selector.border.color
-    };
-
-    &:hover{
-      ${
-        !disabled
-          ? `border-color: ${dropdown.hover.selector.border.color};`
-          : ''
-      };
-    }
-    ${
-      disabled
-        ? `border-color: ${dropdown.disabled.selector.border.color};`
-        : ''
-    };
-    ${
-      selected
-        ? `border-color: ${dropdown.selected.selector.border.color};`
-        : ''
-    };
-    ${
-      isOpen && !disabled
-        ? `border-color: ${dropdown.hover.selector.border.color};`
-        : ''
-    }
+    font-size: ${input.label.font.size.typed}px;
+    color: ${dropdown.input.label.color};
   `}
 `;
 
-const Input = styled.input`
+const Selector = styled.div`
   ${({
+    isOpen,
     disabled,
     selected,
+    error,
     theme: {
       yoga: {
-        baseFont,
+        colors,
         components: { dropdown },
       },
     },
-  }) => `
-    width: 100%;
-    padding: 0;
-
-    background-color: transparent;
-    border: none;
-
-    font-family: ${baseFont.family};
-    font-size: ${dropdown.input.font.size}px;
-    line-height: ${dropdown.input.font.lineHeight}px;
-    cursor: ${disabled ? 'not-allowed' : 'pointer'};
+  }) => css`
+    ${!disabled
+      ? `
+          fieldset {
+            ${
+              isOpen || selected
+                ? `border-color: ${dropdown.hover.selector.border.color};`
+                : ''
+            }
+
+            ${
+              error && !isOpen
+                ? `border-color: ${colors.feedback.attention[1]};`
+                : ''
+            }
+          }
+          &:hover {
+            fieldset {
+              border-color: ${dropdown.hover.selector.border.color};
+            }
+          }
+        `
+      : ''}
+  `}
+`;
 
-    &, &::placeholder {
-      color: ${dropdown.input.font.color};
-      ${selected ? `color: ${dropdown.selected.input.font.color};` : ''}
-      ${disabled ? `color: ${dropdown.disabled.input.font.color};` : ''}
+const Field = styled(Input)`
+  ${({
+    value,
+    isOpen,
+    disabled,
+    theme: {
+      yoga: { transition },
+    },
+  }) => css`
+    &:focus {
+      & ~ legend {
+        max-width: max-content;
+        transition-property: max-width;
+        transition-duration: ${transition.duration[1]}ms;
+      }
+
+      & ~ label {
+        ${labelTransition}
+      }
     }
+
+    ${(value || isOpen) && !disabled
+      ? css`
+          & ~ legend {
+            max-width: max-content;
```

Comment:
Should these styles be replicated since we're already using Yoga's Input that already contains that?